package/MDAnalysis/analysis/secondary_structure/dssp.py

```diff
@@ -0,0 +1,254 @@
+# -*- Mode: python; tab-width: 4; indent-tabs-mode:nil; coding:utf-8 -*-
+# vim: tabstop=4 expandtab shiftwidth=4 softtabstop=4
+#
+# MDAnalysis --- https://www.mdanalysis.org
+# Copyright (c) 2006-2020 The MDAnalysis Development Team and contributors
+# (see the file AUTHORS for the full list of names)
+#
+# Released under the GNU Public Licence, v2 or any higher version
+#
+# Please cite your use of MDAnalysis in published work:
+#
+# R. J. Gowers, M. Linke, J. Barnoud, T. J. E. Reddy, M. N. Melo, S. L. Seyler,
+# D. L. Dotson, J. Domanski, S. Buchoux, I. M. Kenney, and O. Beckstein.
+# MDAnalysis: A Python package for the rapid analysis of molecular dynamics
+# simulations. In S. Benthall and S. Rostrup editors, Proceedings of the 15th
+# Python in Science Conference, pages 102-109, Austin, TX, 2016. SciPy.
+# doi: 10.25080/majora-629e541a-00e
+#
+# N. Michaud-Agrawal, E. J. Denning, T. B. Woolf, and O. Beckstein.
+# MDAnalysis: A Toolkit for the Analysis of Molecular Dynamics Simulations.
+# J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
+#
+
+"""Secondary structure analysis --- :mod:`MDAnalysis.analysis.secondary_structure.dssp`
+=======================================================================================
+
+:Authors: Lily Wang
+:Year: 2020
+:Copyright: GNU Public License v3
+
+.. versionadded:: 1.0.0
+
+This module contains a class to compute secondary structure using the DSSP
+algorithm [Kabsch1983]_, as implemented by MDTraj_ [McGibbon2015]_ .
+
+
+Classes
+-------
+
+.. autoclass:: DSSP
+    :members:
+
+"""
+
+from __future__ import absolute_import, division
+
+import numpy as np
+
+from ...due import due, Doi
+from .base import SecondaryStructureBase
+
+due.cite(Doi("10.1016/j.bpj.2015.08.015"),
+         description="MDTraj",
+         path="MDAnalysis.analysis.secondary_structure.dssp",
+         cite_module=True)
+
+due.cite(Doi("10.1002/bip.360221211"),
+         description="DSSP",
+         path="MDAnalysis.analysis.secondary_structure.dssp",
+         cite_module=True)
+
+due.cite(Doi("10.1093/nar/gku1028"),
+         description="DSSP (new)",
+         path="MDAnalysis.analysis.secondary_structure.dssp",
+         cite_module=True)
+
+
+class DSSP(SecondaryStructureBase):
```

Comment:
I would change the name to DSSPmdtraj or something else. You can then still derive `DSSP(DSSPmdtraj)` but in case at some point we write our own version or bundle it then we can easily switch DSSP over.